jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -838,6 +875,31 @@ public void testGetJavaContainerBuilderWithBaseImage_java12NoBaseImage()
     }
   }
 
+  @Test
+  public void testGetValidPlatformsList() throws InvalidPlatformConfigurationException {
+    Mockito.<List<? extends PlatformConfiguration>>when(rawConfiguration.getPlatforms())
+        .thenReturn(Arrays.asList(new PlatformParametersTest("testArchitecture", "testOs")));
+
+    Assert.assertEquals(
+        ImmutableSet.of(new Platform("testArchitecture", "testOs")),
+        PluginConfigurationProcessor.getPlatformsSet(rawConfiguration));
+  }
+
+  @Test
+  public void testInvalidPlatformsList() throws InvalidPlatformConfigurationException {
```

Comment:
testGetPlatformsSet_osMissing()

That said, I'd add testGetPlatformsSet_architectureMissing() as well.

Reply:
Done

Reply:
Oh, I just noticed that you can remove the `throws InvalidPlatformConfigurationException` clause from the method signature.

Reply:
The moment I remove it , the file raises the 'unhandled exception error `

Reply:
With the latest commit, remove it from `testGetPlatformsSet_architectureMissing` and `testGetPlatformsSet_osMissing`.